Sources/_TestingInternals/include/Includes.h

```diff
@@ -80,6 +80,10 @@
 #include <pthread.h>
 #endif
 
+#if __has_include(<pthread_np.h>)
+#include <pthread_np.h>
```

Comment:
I assume this header contains `posix_spawn_file_actions_addclosefrom_np()`?

Reply:
yup